### DISCUSSION TITLE
[Beta projects] Ability to customize cards

### DISCUSSION THREAD

**Discussion by e0d** [community/community#6503] [Projects and Issues]
I'd like an enhanced ability to customize cards.  Currently they're a bit of a "wall of same" on large projects.

I'd like to be able to add things like custom fields, age in status, covers, images, etc.

**Comment by e0d** [comment]
I've discovered that you have some ability to customize cards, but you can only do that by editing the columns in the grid view.  This is definitely better than nothing.

However, I'd like to be able to add color differentiation to custom fields created from the grid view.  For example:

![image](https://user-images.githubusercontent.com/1270274/138001449-89c05dbf-9b98-4e8b-9489-c85993623674.png)

If you could style enumerated custom field value as you can labels, that would be super valuable.

I'd prefer to create a custom field for team assignments, but would prefer to have the indicators colorized for differentiation.

For clarity, I'd like to colorize the P0, P1 indicators as the label are in the included image.

**Comment by johncrim** [comment]
I'd be happy with just displaying custom fields on cards. For example, we have a "Size" custom field for story points, and it would be super-useful to be able to display that field on the cards in the board view.

**Reply by johncrim** [reply]
Whoa, I guess I didn't look hard enough for this. Looks like the support is already there:
<img width="242" alt="image" src="https://user-images.githubusercontent.com/3312127/157958534-d4500d88-0d94-4669-ae59-b1f7f56b22fb.png">
